Ionic3.gitignore

```diff
@@ -0,0 +1,58 @@
+# Log and tmp data
+*~
+*.sw[mnpcod]
+*.log
+*.lock
+*.tmp
+*.tmp.*
+log.txt
+
+# NPM (Node Package Manager) files
+npm-debug.log*
+node_modules/
+tmp/
+temp/
+.package.tmp.json
+
+# Preprocessors cache files
+.idea/
+.sourcemaps/
+.vscode/
+.sass-cache/
+.versions/
+coverage/
+dist/
+.tmp/
+hooks/
+
+# Ionic 3 plugins files
+platforms/
+plugins/
+plugins/android.json
+plugins/ios.json
+www/
+
+# Trash data
```

Comment:
These rules are covered by existing OS-specific templates under the `Global` folder, and I want to avoid the maintenance work that comes with duplicating